recipe/conda_build_config.yaml

```diff
@@ -0,0 +1,4 @@
+MACOSX_DEPLOYMENT_TARGET:  # [osx and x86_64]
+  - "10.12"                # [osx and x86_64]
+MACOSX_SDK_VERSION:        # [osx and x86_64]
+  - "10.12"                # [osx and x86_64]
```

Comment:
Please see [these docs for context]( https://conda-forge.org/docs/maintainer/knowledge_base.html#requiring-newer-macos-sdks )